foundation/src/main/java/com/decathlon/vitamin/compose/foundation/Shape.kt

```diff
@@ -5,7 +5,7 @@ import androidx.compose.material.Shapes
 import androidx.compose.ui.unit.dp
 
 val shapes = Shapes(
-    small = RoundedCornerShape(2.dp),
-    medium = RoundedCornerShape(2.dp),
-    large = RoundedCornerShape(2.dp)
+    small = RoundedCornerShape(vtmnRadius.radius100),
+    medium = RoundedCornerShape(vtmnRadius.radius200),
```

Comment:
Any good reason to use radius200 here? The Material2 value is `4.dp`, `radius100` for us. Maybe a better choice?